docs/getting-started/sync-with-bitbucket.md

```diff
@@ -7,63 +7,115 @@ slug: /sync-with-bitbucket
+Amplication's Sync With Bitbucket feature is **only available for users on an Enterprise plan**.
+
+Please see: [Differences between sync with a Git provider in Free edition and Pro/Enterprise Plans](/sync-with-git-differences-between-plans)
 ::: 
 
-## Creating a new repository on Bitbucket
+This page provides you with everything you need to work with Bitbucket.
+You'll learn how to do the following:
 
-If you want to create a new repository on Bitbucket, start by navigating to your Bitbucket account and selecting the workspace where you'd like the new repository to be created. Once you're in the chosen workspace, simply click on 'Create Repository'. The URL for this action will typically look like this: `https://bitbucket.org/{your-bitbucket-workspace}/workspace/create/repository`
+- [Authorize Bitbucket](#authorize-amplication-to-access-bitbucket)
+- [Create a new Bitbucket repository](#create-a-new-bitbucket-repository)
+- [Connect to a specific Bitbucket repository](#select-your-repository)
+- [Change git providers for a specific service](#change-your-services-connected-git-provider)
+- [Create a new pull request for Bitbucket](#create-a-new-pull-request)
 
-## Working on Local Amplication Server
+Let's dive right in.
 
-If you are hosting a local Amplication server, you should first read [Connect Amplication server to Bitbucket](/docs/running-amplication-platform/connect-server-to-bitbucket.md) to see how to configure the Server to work with a new Bitbucket application.
+## Prerequisites
 
-## Using the Amplication hosted service
+If you're running Amplication locally, first read [Connect Amplication server to Bitbucket](/running-amplication-platform/connect-server-to-github).
+You'll learn how to configure the server to work with a new Bitbucket application.
 
-If you are using the hosted service on [https://app.amplication.com](https://app.amplication.com/), you can continue with this article.
+## Authorize Amplication to access BitBucket
 
-## Authorize Amplication to access your Bitbucket account
+First, go to your project's dashboard.
+You'll need to go to the _Sync with Git Provider_ page.
+There are 3 ways to do this:
 
-Before integrating your application with Bitbucket, you need to authorize Amplication to get access to your Bitbucket account.
+1. Click _Sync with git provider_ on the sidebar.
+2. Click on the _Sync with git provider_ tile on your project's dashboard.
+3. Click _Connect to git_ on the footer at the bottom of the page.
 
-1. Open up an Amplication project.
-2. Click **Connect to git provider** at the sidebar or at the tile of your resource overview 
 ![resource-overview-git-sync.png](./assets/sync-with-git-provider/resource-overview.png)
 
+You're now on the _Sync with Git Provider_ page.
+Proceed with the following steps to connect a GitHub repository to your service:
```

Comment:
@dericksozo typo. should be Bitbucket instead of GitHub

Reply:
@dericksozo this comment was not fixed yet

Reply:
Hi @mulygottlieb, this line and @GreenMachine01's other feedback was addressed in this [specific commit](https://github.com/amplication/docs/pull/308/commits/f2a615202a01fe249e1c0df264b3afbc19fbf059#diff-cab478d95d24c95ec7c2f321571809e6612bdac44d53e1f9836f3492bb4e236c). Did I miss anything?